provider-forums/src/java/org/sakaiproject/archiver/provider/DateComparator.java

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
License header. You should disable this in the eclipse preferences and it wont get added to new classes.